versions.yaml

```diff
@@ -0,0 +1,25 @@
+module-sets:
+  edot-base:
+    version: v0.124.0
```

Comment:
Will we have to regularly release for every upstream version that is released? If so how this will be performed? Upstream Collector has a specific release schedule with round robin assignment, are we going to have specific teams/people handling this?

Reply:
Yes, the plan is to be handled by the SDH "hero" of the `ingest-otel-data` team 

cc @sboomsma